src/reducers/entities/teams.js

```diff
@@ -140,7 +168,7 @@ export default combineReducers({
     // object where every key is the team id and has and object with the team members detail
     myMembers,
 
-    // object where every key is the team id and has a Set of user ids that are members in the team
+    // object where every key is the team id and has an of members in the team where the key is user id
```

Comment:
ok, we just need to be careful when changing these cause when doing a rehydration it could error out and the store might be invalid at some point